contrib/config-example

```diff
@@ -0,0 +1,38 @@
+###
+## Core Lightning Configuration
+###
+## Configuration manual:
+## man lightningd config 
+###
+
+# Node Name / Color
+alias=<NODE-ALIAS>
+rgb=FFA500
+
+# Run lightningd as a daemon. (Recommended)
+daemon
+
+# Channel Options (Set to defaults, please read the manual)
+fee-base=1000
+fee-per-satoshi=10
+min-capacity-sat=10000
+htlc-minimum-msat=0
+funding-confirms=1
+
+# Bitcoin Settings
+network=bitcoin
```

Comment:
```suggestion
# if you declared a top level config file the network is required otherwise, it
# if the file is inside a network directory like `.lightning/bitcoin` this field can be omitted
network=bitcoin
```

Something like that! 

In addition, I guess we should document what these fields does?